core/field_label.js

```diff
@@ -74,6 +87,13 @@ Blockly.FieldLabel.fromJson = function(options) {
  */
 Blockly.FieldLabel.prototype.EDITABLE = false;
 
+/**
+ * The css class name to apply to the field's textElement_.
+ * @type {?string}
+ * @private
+ */
+Blockly.FieldLabel.prototype.class_ = null;
```

Comment:
Same as other PRs, please move to the constructor.

Reply:
Fixed, I think. See [comment on checkbox pr](https://github.com/google/blockly/pull/2876#discussion_r316258252).